pending-documentation.md

```diff
@@ -375,6 +375,28 @@ In the example above, we declare two productions `~Int_` and `_^%Int__`, and tel
 Set variables are currently only supported by the Haskell backend.
 The use of rules with set variables should be sound for all other backends which just execute by rewriting, however it might not be safe for backends which want to guarantee coverage.
 
+## Debugging
+
+The LLVM Backend has support for integration with GDB. You can run the debugger on a particular program by passing the `--debugger` flag to krun, or by invoking the llvm backend interpreter directly. Below we provide a simple tutorial to explain some of the basic commands supported by the LLVM backend.
```

Comment:
Here you say "Below we provide a simple tutorial", but we seem to be missing the actual tutorial part.

I'm wondering if commits got mixed up between this PR updating the dependencies and another with the debugger tutorial.

Reply:
it's a work in progress. I can factor out this part of the commit if you want? I just needed to get this merged because it contained necessary fixes and I hadn't finished writing the tutorial yet. But I was planning on finishing this in a followup PR. How would you like me to handle it?